### DISCUSSION TITLE
Multipack Talks (Alcohol Free) - now in Github

### DISCUSSION THREAD

**Discussion by si** [multipack/website#47] [Announcements]
After digging through some of the old Basecamp discussions the other day (thanks to a reminder to keep the account active), it reminded me of all those fascinating conversations we had as a community; organising events, working through some technical challenges, just helping each other like a good community should.

Then I realised Github had the feature available. 

With a renewed passion to meet up again (planned for 6 April 2023 at The Old Joint Stocks), and struggling to find the most suitable place on the Interwebs to chat (Twitter is dead, Mastodon is not available to all or easy for group DMs), this might help.

Join the conversation. Suggest improvements or alternatives - but let's get the old gang back together again. It's been _WAY_ too long.

**Comment by paulrobertlloyd** [comment]
Good idea! So many different places to hold discussions, each with their pros and cons but GitHub Discussions is possibly the closest to the way our old school forum worked. Let's give it a shot!
